frontend/src/index.tsx

```diff
@@ -36,9 +40,34 @@ declare global {
 (async () => {
   window.deckyAuthToken = await fetch('http://127.0.0.1:1337/auth/token').then((r) => r.text());
 
+  i18n
+    .use(Backend)
+    .use(LanguageDetector)
+    .use(initReactI18next)
+    .init({
+      load: 'currentOnly',
+      detection: {
+        order: ['querystring', 'navigator'],
+        lookupQuerystring: 'lng',
+      },
+      //debug: true,
+      fallbackLng: 'en-US',
+      interpolation: {
+        escapeValue: false,
+      },
+      backend: {
+        loadPath: 'http://127.0.0.1:1337/locales/{{lng}}.json',
+        customHeaders: {
+          Authentication: window.deckyAuthToken,
```

Comment:
you dont need this if you added it to the ignore list in helpers.py, choose to require auth or remove it from there

Reply:
Removed the bypass on loader.py